CMakeLists.txt

```diff
@@ -132,6 +132,7 @@ if(ROBOTOLOGY_ENABLE_DYNAMICS)
   endif()
   if(ROBOTOLOGY_USES_MATLAB)
     if(NOT ROBOTOLOGY_NOT_USE_SIMULINK)
+      set(ROBOTOLOGY_USES_SIMULINK TRUE)
```

Comment:
The rationale for this is described in the readme at https://github.com/robotology/robotology-superbuild/blob/98500f5f57fff4d14044379e0edd4a8f1d3bbd33/README.md#system-dependencies-7 . Basically, based on my limited experience the vast majority of our users have Matlab+Simulink installed on their computer, so we just require them to set `ROBOTOLOGY_USES_MATLAB` to `TRUE` to enable both software that requires MATLAB and software that requires Simulink. There are (a few, fortunatly) cases in which the user has matlab but no Simulink: in that case, the user select the additional option `ROBOTOLOGY_NOT_USE_SIMULINK` (that default to false) to avoid compiling the software that depends on Simulink. 
I don't think that adding a `ROBOTOLOGY_USES_SIMULINK` and requiring all users to set it to `ON` manually in addition to `ROBOTOLOGY_USES_MATLAB` would be helpful.

See https://github.com/robotology/codyco-superbuild/issues/65#issuecomment-222383441 for when the option was added. The r

Reply:
@gabrielenava I add to Silvio's reply that due to limitations of the `cmakeif` command that you can find in the scripts, specifying a condition (e.g. `cmakeif NOT MYVAR`) is not supported. After `cmakeif` there must be a single cmake variable that can be either `TRUE` or `FALSE`. For this reason I had to introduce this extra `ROBOTOLOGY_USES_SIMULINK` variable.